plugins/modules/azure_rm_privateendpointdnszonegroup.py

```diff
@@ -0,0 +1,351 @@
+                            description:
+                                - IP addresses for the record.
+                            returned: always
+                            type: list
+                            sample: ['10.1.0.4']
+                        provisioning_state:
+                            description:
+                                - Provisioning state of the resource.
+                            returned: always
+                            type: str
+                            sample: Succeeded
+                        record_set_name:
+                            description:
+                                - Name of the record.
+                            returned: always
+                            type: str
+                            sample: myPostgreSqlSrv-123
+                        record_type:
+                            description:
+                                - Type of record.
+                            returned: always
+                            type: str
+                            sample: A
+                        ttl:
+                            description:
+                                - Time to live value of the record.
+                            returned: always
+                            type: int
+                            sample: 10
+        provisioning_state:
+            description:
+                - Provisioning state of the resource.
+            returned: always
+            type: str
+            sample: Succeeded
+'''
+
+try:
+    from msrestazure.tools import resource_id
+    from msrest.polling import LROPoller
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common_ext import AzureRMModuleBaseExt
+
+
+private_dns_zone_configs_spec = dict(
+    name=dict(type="str"),
+    private_dns_zone=dict(type="str")
+)
+
+
+class Actions:
+    NoAction, Create, Update, Delete = range(4)
+
+
+class AzureRMPrivateEndpointDnsZoneGroup(AzureRMModuleBaseExt):
+
+    def __init__(self):
+
+        self.module_arg_spec = dict(
+            name=dict(type="str", required=True),
+            private_endpoint=dict(type="str", required=True),
+            resource_group=dict(type="str", required=True),
+            private_dns_zone_configs=dict(type="list", elements="dict", options=private_dns_zone_configs_spec),
+            state=dict(type="str", default="present", choices=["present", "absent"]),
+        )
+
+        self.name = None
+        self.private_endpoint = None
+        self.resource_group = None
+        self.state = None
+        self.parameters = dict()
+        self.results = dict(
+            changed=False,
+            state=dict()
+        )
+        self.to_do = Actions.NoAction
+
+        super(AzureRMPrivateEndpointDnsZoneGroup, self).__init__(self.module_arg_spec,
+                                                                 supports_tags=False,
```

Comment:
Why do you inherit tags in your document, but set false here in argc? It is recommended that you add it here. And make the following changes. Thank you very much!
```suggestion
                                                                 supports_tags=True,
```